README.md

```diff
@@ -1 +1,198 @@
-# to_do_list
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To-Do list <a name="about-project"></a>
+
+- This website shows a sneak peek into the projects I have developed, showcasing my skills in HTML, CSS, javascript and webpack. 
+
+This is a simple daily activities website that shows what goes on withing a particular day. It has 'add task section' where you can add new task.
+A 'form section' where your added task displys and 'clear section' for end of daily activities deletion.
+
+With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- linter
+- webpack
+- HTML validation
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo
```

Comment:
- [ ] [Optional] This is you last review so I am providing it optional please remove this live demo completely.
### Things to remove
![his](https://github.com/albertkantwi/to_do_list/assets/81727988/67d8e2b9-d765-4fa3-8950-60b0ec3785c5)